### PR TITLE
Spaceship

### PR DESCRIPTION
**Pull request by Debajyati**
## What I did

I was first thinking to create a spiral animation but after creating the layout, (not the keyframes were written till then), I realised the thing I created doesn't resemble a spiral but highly resemble a spaceship! So made it look more like it. Also applied the animation to be the motion of the ship. It rotates, stops and instantly transports to some other point and the motion animation runs forever. I hope it will be liked! ;) 

## Pull request checklist

- [x] This pull request uses **HTML and CSS** only and provides a `meta.json` file.
- [x] All files in this pull request are in a directory named: `<github_username>-<art_name>` without the <>.
- [x] Your pull request contains `index.html` and `styles.css`.
	- [x] Your html file is named exactly: `index.html`
	- [x] Your css file is named exactly: `styles.css`
- [x] Your pull request your `meta.json` with:
	- [x] `githubHandle`: being your unique github user name,
	- [x] `artName` : name of your animation
- [x] Your submission includes **at least one animation**.
- [x] Your pull request **does not modify any other files** in the repository.

#### Not normally needed
- [ ] Check this if you would like to discuss an improvement besides adding animations

## What to expect

- If your pull request meets the checklist requirements it will be approved and merged by a maintainer
- If your pull request needs some changes. A maintainer will leave a comment requesting changes
- If your pull request significantly violates the criteria in the checklist it will be closed with feedback.

### CODE REVIEW

File: Art/Debajyati-SPACESHIP/index.html
Comment (LaurelineP) [inline Art/Debajyati-SPACESHIP/index.html:1]: ## Unmecessary script
- [ ] Please remove the unnecessary script tag from your html

<script src="script.js"></script>

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha :) 
Please address the feedbacks so we can merge your PR

File: Art/Debajyati-SPACESHIP/index.html
Comment (Debajyati) [reply Art/Debajyati-SPACESHIP/index.html:1]: Done!

**Comment by Debajyati** [comment]
Kindly review the changes! I think I have done it right.

**Pull request merged**